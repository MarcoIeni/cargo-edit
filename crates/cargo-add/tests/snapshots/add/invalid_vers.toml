bin.name = "cargo-add"
args = ["add", "my-package@invalid version string"]
status.code = 1
stdout = ""
stderr = """
Error: Invalid version requirement `invalid version string`

Caused by:
    unexpected character 'i' while parsing major version number
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
