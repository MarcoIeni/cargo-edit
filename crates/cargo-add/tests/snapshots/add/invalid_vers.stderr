    Updating '[ROOTURL]/registry' index
Error: Invalid version requirement `invalid version string`

Caused by:
    unexpected character 'i' while parsing major version number
